.github/workflows/ci.yml

```diff
@@ -92,13 +88,18 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ github.job }}-
       - run: |
-          ./gradlew clean uploadArchives --no-daemon --no-parallel --no-configuration-cache
+          ./gradlew build -x lint -x lintVitalRelease
```

Comment:
Maybe a comment or `TODO` pointing to the issue to remember fixing it in the future